sdk/Sdk/FunctionMetadataGenerator.cs

```diff
@@ -185,12 +186,26 @@ private bool TryCreateFunctionMetadata(MethodDefinition method, out SdkFunctionM
                     string assemblyFileName = Path.GetFileName(declaringType.Module.FileName);
 
                     function = CreateSdkFunctionMetadata(functionName, actualMethodName, declaringTypeName, assemblyFileName);
-
-                    return true;
+                }
+                else if (string.Equals(attribute.AttributeType.FullName, Constants.FixedDelayRetryAttributeType, StringComparison.Ordinal) ||
```

Comment:
Should we mark the classes(_FixedDelayRetryAttribute, ExponentialBackoffRetryAttribute_) as `sealed` ? If someone subclass the `FixedDelayRetryAttribute` type, Won't this code fail?